Makefile

```diff
@@ -2,12 +2,12 @@
 PNAME = bfcl
 OBJS = $(PNAME).o ocl_util.o utils.o sha1_16.o aes_128.o ocl_test.o ocl_brute.o
 CFLAGS += -std=c11 -Wall -Werror -O2 -mrdrnd -I$(INTELOCLSDKROOT)/include
-LDFLAGS += -L$(INTELOCLSDKROOT)/lib/x64
+LDFLAGS += -L/usr/lib/x86_64-linux-gnu
```

Comment:
This line should be changed back to the way it originally was since (from what I can tell) it’s needed for Windows users who are using Intel’s OpenCL SDK. As stated before, Linux already looks in `/usr/lib/x86_64-linux-gnu/` for libOpenCL and libmbedcrypto. In addition, MinGW-w64 looks for the two libraries in `mingw64/lib/` by default, but since Intel’s OpenCL SDK won’t be installed there, the compilation will fail.